### PR TITLE
Initial blank controller for Resource Access Manager

### PR DESCRIPTION
**Pull request by michaelhtm**
Issue https://github.com/aws-controllers-k8s/community/issues/492

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: metadata.yaml
Comment (a-hilaly) [inline metadata.yaml:1]: we need to fill this file

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# See the OWNERS docs at https://go.k8s.io/owners#owners_aliases
+
+aliases:
+  core-ack-team:
+    - michaelhtm
```
Comment (a-hilaly) [inline OWNERS_ALIASES:5]: Can we have the same core team as the one in test-infra

File: ATTRIBUTION.md
Comment (a-hilaly) [inline ATTRIBUTION.md:1]: i was going to comment here :) did you generate this file using attribution-gen?

Comment (michaelhtm) [reply ATTRIBUTION.md:1]: I did. Pretty cool tool 🔥

File: OWNERS_ALIASES (hunk 1 above)
Comment (michaelhtm) [reply OWNERS_ALIASES:5]: when i tried it ealier prow didn't like it..trying again

Comment (a-hilaly) [reply OWNERS_ALIASES:5]: It's ok as long as we know it's correct.

Comment (a-hilaly) [reply OWNERS_ALIASES]: let's go with only the core members for now, we can sync the emeritus once we figure out how to do it properly.

**Review by a-hilaly — APPROVED** [review]
👍

**Pull request merged**